### PR TITLE
cmd/snap-confine/tests: fix shellcheck on recently added files

### PR DESCRIPTION
**Pull request by zyga**
Shellcheck is not tested automatically becauseof packaging issues related to
that. This has lead to an issue where we merged something that fails (harmless)
shellcheck verification and is a problem for downstream packaging.

Signed-off-by: Zygmunt Krynicki <zygmunt.krynicki@canonical.com>

### CODE REVIEW

File: interfaces/builtin/background.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
```
Comment (morphis) [inline interfaces/builtin/background.go]: This file looks like it doesn't belong into this PR.

File: cmd/snap-confine/tests/common.sh [hunk 2]
```diff
@@ -60,7 +60,7 @@ SHM="$(mktemp -d -p /run/shm)"
 trap 'rm -rf $TMP $SHM' EXIT
 
 # name snap-confine as the test name for improved logging
-L="$TMP/`basename $0`"
+L="$TMP/$(basename "$0")"
```
Comment (ogra1) [inline cmd/snap-confine/tests/common.sh]: only do:
L=$TMP/$(basename "$0")

there shoudl be no need for the outer quotes that makes it more confusing to read...

File: interfaces/builtin/background.go (hunk 1 above)
Comment (zyga) [inline interfaces/builtin/background.go]: Oh, how the hell :D

File: cmd/snap-confine/tests/common.sh (hunk 2 above)
Comment (zyga) [inline cmd/snap-confine/tests/common.sh]: I think shellcheck disagrees.

**Comment by morphis** [comment]
LGTM

**Comment by morphis** [comment]
Verified this fixes the build issues on Fedora.

**Pull request merged**